apps/core-app/src/renderer/src/modules/box/adapter/hooks/useClipboard.ts

```
@@ -250,52 +219,33 @@ export function useClipboard(
 
   async function applyToActiveApp(item?: IClipboardItem): Promise<boolean> {
     const target = item ?? clipboardOptions.last
-    if (!target)
-      return false
+    if (!target) return false
```

ESLint

[error] Expect newline after if

(antfu/if-newline)